src/main/java/frc/robot/commands/IndicatorLights.java

```diff
@@ -15,9 +15,10 @@ public class IndicatorLights extends Command {
  
   Lights lights;
  /**
- * This command controls the lights.  Section One Lights: If a note is seen, lights = dark orange-brown. Otherwise,
- *  lights = oddly deep dark purple.  Section Two Lights: If the shooter is in range, and is ready, lights = dark
- * green. If it is in range, but not ready, lights = a dark green-yellow-brown. If neither is true, lights = deep dark purple
+ * This command controls the lights.  Middle Lights: If a note is seen, lights = dark orange-brown. Otherwise,
+ *  lights = purple. 
+ * Side Lights: If the shooter is in range, and is ready, lights = dark green. 
```

Comment:
The colors in this comment don't seem to be correct-- the lights are never purple.